migrate/20230731_usage_records.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    create_table(:usage_record) do
+      column :id, :uuid, primary_key: true, default: nil
+      foreign_key :project_id, :project, type: :uuid, null: false
+      column :resource_id, :uuid, null: false
+      foreign_key :initiated_by, :accounts, type: :uuid, null: false
+      foreign_key :terminated_by, :accounts, type: :uuid
```

Comment:
If a user created VM, then deleted VM and account. Do we want to keep usage_record in DB for a time? It's same same for project too. What's happens when project deleted?

Reply:
I changed the project deletion to be soft delete and denormalized resource name